src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/ConfigBackup.xml

```diff
@@ -0,0 +1,176 @@
+﻿<configuration>
+  <!-- 
+    This is a copy of the old app.config for reference.
+    My intention is to replace all the XML config with code.
```

Comment:
It sounds reasonable. The reason why we did this in config to begin with was so that it'd be easier to maintain with multiple people trying to modify a single test server. Now that the code is committed into the repo and the server is running on developer machines, I see few benefits of keeping server configuration in config.